.travis.yml

```diff
@@ -6,6 +6,9 @@ addons:
       libow-dev
       openssh-server
       openssh-client
+      libudev-dev
+      libusb-1.0-0-dev
+      libfox-1.6-dev
```

Comment:
We have since switched to github workflows, since travis and opensource apparently don't agree any longer.